owncloudData/src/main/java/com/owncloud/android/data/files/db/OCFileEntity.kt

```diff
@@ -127,5 +139,8 @@ data class OCFileEntity(
         private fun Cursor.getStringFromColumnOrEmpty(
             columnName: String
         ): String = getColumnIndex(columnName).takeUnless { it < 0 }?.let { getString(it) }.orEmpty()
+
+        const val FILE_OWNER = "owner"
+        const val FILE_SPACE_ID = "spaceId"
```

Comment:
In this case, having those names as constants are not that useful. For spaces, we use them as field name on the database table, but here we could remove the constant and use it directly

Reply:
I think they may be useful if in a future more classes use this name, rather than having hard-coded strings. Anyways, I moved it to ProviderMeta, I think it fits better there.